lib/NIF/NifFile.h

```diff
@@ -12,6 +12,7 @@ struct OptOptions {
 	bool headParts = false;
 	bool removeParallax = true;
 	bool calcBounds = true;
+    bool bsTriShape = false;
```

Comment:
Default should be true to have it work the same way as before by default if the new property isn't changed.

Reply:
Agreed.